src/Integration/MefServices/ActiveSolutionBoundTracker.cs

```diff
@@ -52,13 +53,19 @@ internal sealed class ActiveSolutionBoundTracker : IActiveSolutionBoundTracker,
         private readonly ILogger logger;
         private readonly uint boundSolutionContextCookie;
 
+        private readonly IEnumerable<IBoundSolutionObserver> boundSolutionObservers;
+        private readonly IGitEvents gitEvents;
+
         public event EventHandler<ActiveSolutionBindingEventArgs> SolutionBindingChanged;
         public event EventHandler SolutionBindingUpdated;
 
         public BindingConfiguration CurrentConfiguration { get; private set; }
 
         [ImportingConstructor]
-        public ActiveSolutionBoundTracker(IHost host, IActiveSolutionTracker activeSolutionTracker, ILogger logger)
+        public ActiveSolutionBoundTracker(IHost host, IActiveSolutionTracker activeSolutionTracker,
```

Comment:
The ASBT imports a list of things that need to be notified about solution changes before the events are raised.

The server branch mapper and git monitor both implement this interface.